registry/storage/driver/testsuites/testsuites.go

```diff
@@ -1169,8 +1169,8 @@ var randomBytes = make([]byte, 128<<20)
 
 func init() {
 	// increase the random bytes to the required maximum
-	for i := range randomBytes {
-		randomBytes[i] = byte(rand.Intn(2 << 8))
+	if _, err := io.ReadFull(rand.Rand, randomBytes); err != nil {
```

Comment:
`rand.Rand` is the type, needs to be created through `rand.New(rand.NewSource(3))` (note 3 is an arbitrary seed, I like having set seeds for unit tests though). It is unfortunate `math/rand` doesn't give access to its global rand reader.